setup/gcp/installer/util/constants.py

```diff
@@ -288,6 +288,11 @@ class DeploymentStatus(Enum):
     'https://cloud.google.com/resource-manager/docs/'
     'creating-managing-organization')
 
+MESSAGE_RUN_FREQUENCY = (
+    'Forseti will run once every 8 hours, you can update the run '
+    'frequency in the server deployment template field "run-frequency"'
+    ' or edit the cron job scheduled on the server VM directly.')
```

Comment:
I don't think that we should suggest `OR` here when editing the cron job directly on the VM.  Because it will get overwritten on update or reset.  What else would be cool?  Should we recommend something like update the deployment template, and then do a gcloud update of the server?

Reply:
Updated.